modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormDisplayContext.java

```diff
@@ -548,10 +548,8 @@ public String getSuccessPageDescription(Locale locale)
 
 		LocalizedValue body = ddmFormSuccessPageSettings.getBody();
 
-		return HtmlUtil.escape(
-			GetterUtil.getString(
-				body.getString(locale),
-				body.getString(body.getDefaultLocale())));
+		return GetterUtil.getString(
+			body.getString(locale), body.getString(body.getDefaultLocale()));
```

Comment:
@rijema the `HtmlUtil` was added to fix [LPSA-56622](https://liferay.atlassian.net/browse/LPSA-56622) (see [commit](https://github.com/liferay-forms/liferay-portal/commit/52ac0e4f2471c0bf41ac494e5c8c4557bde46bae#diff-f5aa144644d14758bc53c5468b867b46ffc159b0bb95359cb2b26990a448bf4aR48)). 

Please check if you are not causing a regression by adding the `'"><img src=x onerror=alert(1)>` value to the Success Page Description.

Reply:
Hi @carolmariaabb , thanks! Silly me. I concluded that if all the function was changed by Rodrigo, it wasn't a security ticket. I didn't get that far. [Screencast from 2024-04-17 07-48-31.webm](https://github.com/liferay-forms/liferay-portal/assets/41641596/d9ebc780-e23f-46cb-893d-f3b4e5da05d3)

Reply:
@rijema so it's working fine?

Reply:
Yes! I didn't receive any alert in the screen after submitting the forms and rendering the success page.